README.md

```diff
@@ -125,68 +191,48 @@ cratedb-mcp serve --transport=streamable-http
 Alternatively, use the `CRATEDB_MCP_TRANSPORT` environment variable instead of
 the `--transport` option.
 
-### Anthropic Claude
-To use the MCP version within Claude Desktop, you can use the following configuration:
+### Use
 
-```json
-{
-  "mcpServers": {
-    "my_cratedb": {
-      "command": "uvx",
-      "args": ["cratedb-mcp", "serve"],
-      "env": {
-        "CRATEDB_CLUSTER_URL": "http://localhost:4200/",
-        "CRATEDB_MCP_TRANSPORT": "stdio"
-      }
-    }
-  }
-}
-```
+To connect to the MCP server using any of the available [MCP clients], use one
+of the AI assistant applications, or refer to the programs in the [examples folder].
 
-### Dry-run
+We collected a few example questions that have been tested and validated by
+the team, so you may also want to try them to get started. Please remember
+that LLMs can still hallucinate and give incorrect answers.
 
-You can use [mcptools], a Swiss Army Knife for MCP Servers, to talk to the
-CrateDB MCP Server from the command line. The following operations do not
-require a language model.
+* Optimize this query: "SELECT * FROM movies WHERE release_date > '2012-12-1' AND revenue"
+* Tell me about the health of the cluster
+* What is the storage consumption of my tables, give it in a graph.
+* How can I format a timestamp column to '2019 Jan 21'?
 
-Install software packages.
-```shell
-brew tap f/mcptools
-brew install mcp uv
-```
+Please also explore the [example questions] from another shared collection.
 
-Explore the Text-to-SQL API.
-```shell
-mcpt call query_sql --params '{"query":"SELECT * FROM sys.summits LIMIT 3"}' uvx cratedb-mcp serve
-```
-```shell
-mcpt call get_table_metadata uvx cratedb-mcp serve
-```
-```shell
-mcpt call get_health uvx cratedb-mcp serve
-```
+## Project information
 
-Exercise the documentation server API.
-```shell
-mcpt call get_cratedb_documentation_index uvx cratedb-mcp serve
-```
-```shell
-mcpt call \
-  fetch_cratedb_docs --params '{"link":"https://cratedb.com/docs/cloud/en/latest/_sources/cluster/integrations/mongo-cdc.md.txt"}' \
-  uvx cratedb-mcp serve
-```
+### Acknowledgements
+Kudos to the authors of all the many software components and technologies
+this project is building upon.
 
-## Development
+### Contributing
+The `cratedb-mcp` package is an open-source project, and is [managed on
+GitHub]. Contributions of any kind are welcome and appreciated.
+To learn how to set up a development sandbox, please refer to the
+[development documentation].
 
-To learn how to set up a development sandbox, see the [development documentation](./DEVELOP.md).
+### Status
+The software is in the alpha stage, please expect breaking changes.
+Version pinning is strongly recommended, especially if you use it as a library.
```

Comment:
Maybe not much will "break" in the future, as the interface is very slim. In this spirit, the statement possibly sounds too scary?

Reply:
Rephrased:

> The software is in the alpha stage, so breaking changes may happen.